invenio_theme/assets/semantic-ui/js/invenio_theme/truncate.js

```diff
@@ -0,0 +1,24 @@
+import $ from "jquery";
+
+$(".truncate-button").on("click", function () {
+  const $clickButton = $(this);
+  const siblings = $clickButton.siblings();
```

Comment:
Why this?

Anyway, can we avoid playing with siblings? it is too much relying on the code.
We should have clear elements with classes, e.g.
```
const btnShowLess = $(".truncate-btn-show-less");
const btnShowMore = $(".truncate-btn-show-more");
```
etc for all the code